src/main/java/karuna/karuna_backend/exception/RestExceptionHandler.java

```diff
@@ -1,9 +1,9 @@
-package karuna.karuna_backend.Errors;
```

Comment:
I would move those Global Exception handlers either way, all into exception package, or every single one into packages destinated for exception target, therefore this one should be in user -> infrastructure? Something like that.